ec2/sc-ec2-linux-jumpcloud.yaml

```diff
@@ -295,12 +295,11 @@ Resources:
         TagRootVolume:
           commands:
             01_tag_root_disk:
-              command: !Join
-                - ''
-                - - ". /opt/sage/bin/instance_env_vars.sh;"
-                  - "/usr/bin/aws ec2 create-tags --region $AWS_REGION --resources $ROOT_DISK_ID "
-                  - "--tags Key=Name,Value=$EC2_INSTANCE_ID-root Key=cloudformation:stack-name,Value=$STACK_NAME "
-                  - "Key=Department,Value=$DEPARTMENT Key=Project,Value=$PROJECT Key=OwnerEmail,Value=$OWNER_EMAIL"
+              command: !Sub |
```

Comment:
if you refactor this for one file why not refactor for all files?

Reply:
There are only a few stings like this., and I've repeated it across the templates that have this string. If we want to refactor to simply all the templates, we can make a new ticket about it.